Components/Input/input.css

```diff
@@ -0,0 +1,46 @@
+.input-group {
+    color: var(--default-color);
+    font-size: 1rem;
+    font-weight: 600;
+    line-height: 1.5rem;
+    margin: 10px 1.5rem;
+    width: 80%;
+}
+
+
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+.input-simple {
+    border: 1px solid #999;
+}
+
+input {
+    background: var(--color-white);
+    padding: 10px 11px;
+    float: left;
```

Comment:
Not sure about float, but can it be implemented with either flex or other positioning properties.